src/core/FeedsManager.cpp

```diff
@@ -271,6 +271,8 @@ void Feed::update()
 
 										entry.publicationTime = normalizeTime(entry.publicationTime);
 
+										entry.lastReadTime = existingEntry.lastReadTime;
```

Comment:
Please move this line just before `entry.publicationTime = normalizeTime(entry.publicationTime);`, without adding empty line.

Reply:
@Emdek This is fixed.